crypto/crypto.go

```diff
@@ -34,11 +35,11 @@ func Sign(key SigningKey, message []byte) []byte {
 	return ed25519.Sign(ed25519.PrivateKey(key), message)
 }
 
-func Verify(key SigningKey, message, sig []byte) bool {
-	pk, ok := ed25519.PrivateKey(key).Public().(ed25519.PublicKey)
-	if !ok {
-		return false
-	}
+func SigningKeyToPublic(key SigningKey) []byte {
+	return ed25519.PrivateKey(key).Public().(ed25519.PublicKey)
+}
+
+func Verify(pk, message, sig []byte) bool {
 	return ed25519.Verify(pk, message, sig)
 }
```

Comment:
I had a slightly different suggestion in https://github.com/coniks-sys/coniks-go/blob/str-tb-verify-idea/crypto/crypto.go#L17 declaring a type for the public key and changing the signature of ``Verify()`` accordingly. Not sure which approach is better/cleaner.